include/http2.hrl

```diff
@@ -85,8 +85,8 @@
 
 -type stream_id() :: non_neg_integer().
 -record(frame_header, {
-    length      :: non_neg_integer(),
-    type        :: frame_type(),
+    length      :: non_neg_integer() | undefined,
```

Comment:
Can we be explicit to have `length = undefined` when we want to default to `undefined`?